src/metalookup/lib/settings.py

```diff
@@ -19,13 +19,9 @@
 ENABLE_CACHE_CONTROL_ENDPOINTS = os.environ.get("ENABLE_CACHE_CONTROL_ENDPOINTS", "True") == "True"
 CACHE_WARMUP_CONCURRENCY = int(os.environ.get("CACHE_WARMUP_CONCURRENCY", 6))
 
-# Splash
-SPLASH_URL = os.environ.get("SPLASH_URL", "http://splash:8050")
-SPLASH_TIMEOUT = int(os.environ.get("SPLASH_TIMEOUT", 90))
-SPLASH_HEADERS = {
-    "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) "
-    "AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Safari/537.36"
-}
+# Playwright
+PLAYWRIGHT_WS_ENDPOINT = os.environ.get("PLAYWRIGHT_WS_ENDPOINT", "ws://playwright:3000")
+PLAYWRIGHT_PAGE_LOAD_TIMEOUT = int(os.environ.get("PLAYWRIGHT_PAGE_LOAD_TIMEOUT", 15))
```

Comment:
The default 15 would be 15s playwright waits for idle network traffic, right? That seems very large, please reduce, e.g., to the 500ms you mentioned in the comments.

Reply:
Playwright waits for idle network, but at max for PLAYWRIGHT_PAGE_LOAD_TIMEOUT seconds.
I.e. if we set this to 500ms, and the page does not fully load within that time, we get a timeout error and no response from the service. Hence I would remain konservative with a higher timeout value.
For reference: splash always waited at least 10seconds.

Reply:
Aaah, thanks for clarification. Yes, please leave it at 15s.